### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by felixatdlr**
## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,95 @@
...
+
+2. **Navigate to a Subproject:**
+    ```sh
+    cd dataquality
+    ```
+
+3. **Follow the Instructions:**
+   Each subproject contains a README file with detailed instructions on setting up and using the project.
+
+## Contributing
```
Comment (SebastianBezold) [inline README.md]: There is already a `CONTRIBUTING.md` file present. This should be used over a separate description in the `README.md`.

**Review by SebastianBezold — CHANGES_REQUESTED** [review]
Hi @felixatdlr,

Could you please remove the "Contributing" section in favor of the `CONTRIBUTING.md` file and resolve the merge conflicts

File: README.md (hunk 1 above)
Comment (felixatdlr) [reply README.md]: Done. See https://github.com/eclipse-tractusx/emergingtechnologies/pull/6/commits/13a63efe9eb946acaf36cb6922485f8ac0ed2dde

**Review by SebastianBezold — APPROVED** [review]

**Comment by SebastianBezold** [comment]
Hi @felixatdlr could you please fix the merge conflict. I guess you need to rebase because #5 was merged

**Pull request closed without merging**